### DISCUSSION TITLE
Welcome to DSCK-Website Discussions!

### DISCUSSION THREAD

**Discussion by singhayushh** [DSCKGEC/DSCK-Website#17] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anindyamitra15** [comment]
I have a suggestion though, what about making a check list for the tasks left for the website, like what factors do we have to consider in the raw-design (or in some other parts like on the react part or so, ..we may organise a discussion session about that and make a list where the features and TODOs shall be updated. How 'bout doing this?
